scripts/devops_tasks/common_tasks.py

```diff
@@ -32,6 +32,7 @@
 
 DEV_REQ_FILE = "dev_requirements.txt"
 NEW_DEV_REQ_FILE = "new_dev_requirements.txt"
+TRACK2_IDENTIFIER_REQ_PACKAGES = ["azure-core", "azure-mgmt-core"]
```

Comment:
Lets try avoid using the name "track2". Lets use something like NEW_REQ_PACKAGES